src/registry/address/edit-validation.ts

```diff
@@ -0,0 +1,21 @@
+import {IntlShape} from 'react-intl';
+import {z} from 'zod';
+
+import {buildCommonSchema} from '@/registry/validation';
+
+// Constraints taken from the BRK API (apart from postcode which comes from our postcode component)
+const addressSchema = z.object({
+  postcode: z.string().regex(/^[1-9][0-9]{3} ?(?!sa|sd|ss|SA|SD|SS)[a-zA-Z]{2}$/),
+  houseNumber: z.string().regex(/^\d{1,5}$/),
+  houseLetter: z.string().regex(/^[a-zA-Z]$/),
+  houseNumberAddition: z
+    .string()
+    .regex(/^([a-z,A-Z,0-9]){1,4}$/)
+    .optional(),
+});
+
+const defaultValueSchema = z.object({defaultValue: addressSchema});
```

Comment:
this is defined, but not exposed anywhere, so you can now have validation errors that are not rendered anywhere. You can leave it as is, since no inputs also means you can't enter invalid values.

Reply:
Yeah even having the `addressSchema` doesn't seem that relevant as we don't allow default values.